pkg/base-dev/lib/ethereum.hoon

```diff
@@ -942,6 +942,15 @@
   %-  render-hex-bytes
   (as-octs:mimes:html n)
 ::
+++  num-to-hex-minimal
+  |=  n=@
+  ^-  tape
+  %-  prefix-hex
+  ?:  =(0 n)
+    "0"
+  %-  render-hex-bytes-minimal
+  (as-octs:mimes:html n)
```

Comment:
I mean to change it like so:

```suggestion
  ((x-co:co 1) n)
```

And then deleting the `+render-hex-bytes-minimal` arm you added.

Reply:
done.